CookSmart/CookSmart/IngredientListViewController.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  IngredientListViewController.swift
+//  cake
+//
+//  Created by Alex King on 3/28/20.
+//  Copyright © 2020 Olga Galchenko. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+class IngredientListViewController: UIHostingController<IngredientListView> {
+  init() {
+    super.init(rootView: IngredientListView())
+  }
+
+  @objc
+  dynamic required init?(coder _: NSCoder) {
+    assertionFailure("init(coder:) has not been implemented")
+    return nil
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    CSIngredients.sharedInstance()?.refreshRecents()
+  }
+}
+
+struct IngredientListView: View {
+  @State private var searchText: String = ""
+
+  var body: some View {
+    NavigationView {
+      List {
+        SearchBar(text: $searchText)
+        ForEach(CSIngredients.sharedInstance()!.ingredientList) { group in
+          Section(header: Text(group.name)) {
+            ForEach(group.ingredients) { ingredient in
+              IngredientListCell(ingredientName: ingredient.name)
+            }
+          }
+        }
+      }
+      .listStyle(GroupedListStyle())
+      .navigationBarTitle("Ingredients")
+      .navigationBarItems(leading:
+        Button(action: {
+          print("Pressed")
+        }, label: {
+          Image(systemName: "xmark")
+            .foregroundColor(SwiftUI.Color(Color.cakeRed))
+            .font(Font.system(size: 20, weight: .medium, design: .default))
```

Comment:
can we try Avenir here?

Reply:
Wait, why is there font in here at all? This is the closing `X` thing, right?